### PR TITLE
Add support for requiring 2fa to publish specific packages.

### PR DESCRIPTION
**Pull request by iarna**
The nrc PR was: https://github.com/npm/npm-registry-client/pull/175

### CODE REVIEW

File: lib/access.js [hunk 1]
```diff
@@ -61,17 +66,22 @@ function access (args, cb) {
...
-    params.auth = auth
-    try {
-      return npm.registry.access(cmd, uri, params, function (err, data) {
-        if (!err && data) {
-          output(JSON.stringify(data, undefined, 2))
-        }
-        cb(err, data)
+    return Bluebird.try(() => {
+      params.auth = auth
+      try {
```
Comment (iarna) [inline lib/access.js:71]: This is super weird and actually what the API for this function is... it throws if it's given a command its never heard of, but ALSO takes an errback that it uses for network errors. I suppose it's not _that_ unreasonable... it's like an argument assertion.

Just with promises it feels much weirder.

**Comment by ljharb** [comment]
Will this status show up in `npm show`?

**Comment by zkat** [comment]
This stuff is pretty well tested n-r-c side, so I'll just leave it there. We can write tests for it when this gets ported to n-r-f.

This _does_ need to have the `access` tests updated because of new output (see Travis)

**Comment by zkat** [comment]
This is getting superseded by https://github.com/npm/cli/pull/2 so I'm gonna close it.

**Pull request closed without merging**